documents/registerNach.md

```diff
@@ -0,0 +1,645 @@
+    "entity": "invoice",
+    "receipt": "Receipt No. 27",
+    "invoice_number": "Receipt No. 27",
+    "customer_id": "cust_BMB3EwbqnqZ2EI",
+    "customer_details": {
+        "id": "cust_BMB3EwbqnqZ2EI",
+        "name": "Gaurav Kumar",
+        "email": "gaurav.kumar@example.com",
+        "contact": "9123456780",
+        "gstin": null,
+        "billing_address": null,
+        "shipping_address": null,
+        "customer_name": "Gaurav Kumar",
+        "customer_email": "gaurav.kumar@example.com",
+        "customer_contact": "9123456780"
+    },
+    "order_id": "order_FHrZiBOkWHZPOp",
+    "line_items": [],
+    "payment_id": null,
+    "status": "issued",
+    "expire_by": 1647483647,
+    "issued_at": 1595491154,
+    "paid_at": null,
+    "cancelled_at": null,
+    "expired_at": null,
+    "sms_status": "sent",
+    "email_status": "sent",
+    "date": 1595491154,
+    "terms": null,
+    "partial_payment": false,
+    "gross_amount": 0,
+    "tax_amount": 0,
+    "taxable_amount": 0,
+    "amount": 0,
+    "amount_paid": 0,
+    "amount_due": 0,
+    "currency": "INR",
+    "currency_symbol": "₹",
+    "description": "12 p.m. Meals",
+    "notes": {
+        "note_key 1": "Beam me up Scotty",
+        "note_key 2": "Tea. Earl Gray. Hot."
+    },
+    "comment": null,
+    "short_url": "https://rzp.io/i/bzDYbNg",
+    "view_less": true,
+    "billing_start": null,
+    "billing_end": null,
+    "type": "link",
+    "group_taxes_discounts": false,
+    "created_at": 1595491154,
+    "idempotency_key": null,
+    "token": {
+        "method": "nach",
+        "notes": {
+            "note_key 1": "Beam me up Scotty",
+            "note_key 2": "Tea. Earl Gray. Hot."
+        },
+        "recurring_status": null,
+        "failure_reason": null,
+        "currency": "INR",
+        "max_amount": 50000,
+        "auth_type": "physical",
+        "expire_at": 1947483647,
+        "nach": {
+            "create_form": true,
+            "form_reference1": "Recurring Payment for Gaurav Kumar",
+            "form_reference2": "Method Paper NACH",
+            "prefilled_form": "https://rzp.io/i/exdIzYN",
+            "upload_form_url": "https://rzp.io/i/bzDYbNg",
+            "description": "12 p.m. Meals"
+        },
+        "bank_account": {
+            "ifsc": "HDFC0001233",
+            "bank_name": "HDFC Bank",
+            "name": "Gaurav Kumar",
+            "account_number": "11214311215411",
+            "account_type": "savings",
+            "beneficiary_email": "gaurav.kumar@example.com",
+            "beneficiary_mobile": "9123456780"
+        },
+        "first_payment_amount": 0
+    },
+    "nach_form_url": "https://rzp.io/i/exdIzYN"
+}
+```
+-------------------------------------------------------------------------------------------------------
+
+## Create an order to charge the customer
+
+```rb
+param_attr = {
+  "amount": 100,
+  "currency": "INR",
+  "receipt": "Receipt No. 1",
+  "notes": {
+    "key1": "value3",
```

Comment:
payment_capture is missing

Reply:
added 
https://github.com/razorpay/razorpay-ruby/blob/update_readme/documents/registerNach.md?plain=1#L326